src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/service/FlowCacheService.java

```diff
@@ -49,9 +52,8 @@ public FlowCacheService(PersistenceManager persistenceManager, Clock clock,
         this.clock = clock;
         this.flowRttStatsExpirationTime = flowRttStatsExpirationTime;
         this.carrier = carrier;
-
-        FlowRepository flowRepository = persistenceManager.getRepositoryFactory().createFlowRepository();
-        initCache(flowRepository);
+        flowRepository = persistenceManager.getRepositoryFactory().createFlowRepository();
+        activate();
     }
 
     private void initCache(FlowRepository flowRepository) {
```

Comment:
Init cache method has `try catch Expection` block. It means that if something will go wring - init cache just write a message into log, activate() will pass and it will look like every thing is ok, but cache is empty.

Try to use this as reference. https://github.com/telstra/open-kilda/pull/4872/files

There we:
1. activate service in Bolt. 
2. catch exceptions and return false if there are some problems with activation

Reply:
done